Sources/PointFree/LaunchSignup/LaunchSignup.swift

```diff
@@ -25,6 +29,34 @@ private func airtableStuff<I>(_ conn: Conn<I, String>) -> IO<Conn<I, Either<Prel
   return result.map { conn.map(const($0)) }
 }
 
+func notifyUsOfNewSignup<I>(_ conn: Conn<I, String>) -> IO<Conn<I, String>> {
```

Comment:
middleware that takes the incoming email, sends us the notification email, and then passes the email along through the middleware.